tox.ini

```diff
@@ -6,38 +6,33 @@ max-complexity = 20
 
 [tox]
 envlist =
-  py{26,27,py}-flask{08,09,10},py33-flask010,py34-flask010
+  py{27}-flask{10,102},py{34}-flask{10,102},py{35}-flask{10,102},py{36}-flask{10,102}
```

Comment:
What about having `py37` as well?

Reply:
Support for 3.7 seems slightly patchy on Travis currently, but maybe we could configure it using the method shown [here?](https://github.com/travis-ci/travis-ci/issues/9815) I'm not sure if `sudo:false` is required for tox-travis to work or not, was my concern.